src/util/http_utils.js

```diff
@@ -946,6 +946,40 @@ function set_response_headers_from_request(req, res) {
     if (req.query['response-expires']) res.setHeader('Expires', req.query['response-expires']);
 }
 
+/**
+ * autheticate jwt token for prometheus metrics and version request
+ * @param {nb.S3Request} req
+ * @param {nb.S3Response} res
+ * @param {string[]} roles
+ */
+function authorize_bearer(req, res, roles) {
+    let token = req.headers.authorization;
+    try {
+        if (!token) {
+            dbg.error('Missing authorization header : ', token);
+            throw new S3Error(S3Error.AccessDenied);
+        }
+        if (token.includes('Bearer ')) {
```

Comment:
this bearer header format should be required, not optional, if it doesnt have the bearer prefix, thats an error